vendor/assets/stylesheets/dvl/core/forms.scss

```diff
@@ -400,19 +402,50 @@ $styledSelectHeight: 2.25rem;
     &.error + div {
       border-color: $errorColor;
     }
+    &:hover + div {
+      background: #fcfcfc;
+    }
+    &:active + div {
+      background: #f7f7f7;
+    }
+    &[readonly],
+    &[disabled] {
+      + div {
+        opacity: 0.6;
+        background: #f5f5f5;
+      }
+      &:active + div {
+        background: #f5f5f5;
+      }
+    }
   }
 }
 
 .styled_select {
   height: $styledSelectHeight;
+  background: #fff;
+  border: 1px solid #bbb;
+  border-radius: $radius;
+
+}
+
+.styled_select_wrapper,
```

Comment:
Empty rule<br>Selector `styled_select_wrapper` should be written in lowercase with hyphens<br>Selector `dropdown_toggle_button` should be written in lowercase with hyphens